http/sys_health.go

```diff
@@ -148,6 +148,8 @@ func getSysHealth(core *vault.Core, r *http.Request) (int, *HealthResponse, erro
 		code = drSecondaryCode
 	case !perfStandbyOK && perfStandby:
 		code = perfStandbyCode
+	case perfStandbyOK && perfStandby:
+		code = activeCode
```

Comment:
This line is technically a noop. Can you combine these two case statements into just `case perfStandby` and then do an `if` in the case? I think that will be a little cleaner, then the case is just top-level on if it's a perf standby. In fact I'd suggest doing the same to the case below: have it just be `case standby` and an `if` for the standbyOK part.